src/controllers/update/due.diligence.overseas.entity.controller.ts

```diff
@@ -0,0 +1,16 @@
+import { NextFunction, Request, Response } from "express";
+
+import { getDueDiligence, postDueDiligence } from "../../utils/due.diligence";
+import {
+  WHO_IS_MAKING_UPDATE_URL,
+  OVERSEAS_ENTITY_DUE_DILIGENCE_PAGE,
+  OVERSEAS_ENTITY_REVIEW_URL
+} from "../../config";
+
+export const get = (req: Request, res: Response, next: NextFunction) => {
+  getDueDiligence(req, res, next, WHO_IS_MAKING_UPDATE_URL, OVERSEAS_ENTITY_DUE_DILIGENCE_PAGE);
```

Comment:
Is OVERSEAS_ENTITY_DUE_DILIGENCE_PAGE not from the registration journey?

Reply:
fixed